packages/hardhat-core/src/internal/hardhat-network/stack-traces/solidity-stack-trace.ts

```diff
@@ -84,6 +86,13 @@ export interface PanicErrorStackTraceEntry {
   isInvalidOpcodeError: boolean;
 }
 
+export interface CustomErrorStackTraceEntry {
+  type: StackTraceEntryType.CUSTOM_ERROR;
+  message: string;
```

Comment:
What's the message here? The return data?

Reply:
This has the already parsed error message. I'll add a comment explaining that.